bot/moderation/moderation.py

```diff
@@ -22,6 +25,7 @@ def __init__(self, bot):
         self._db_connector = DatabaseConnector(constants.DB_FILE_PATH, constants.DB_INIT_SCRIPT)
 
     @commands.group()
+    @command_log
```

Comment:
I would put `@command_log` always on top. It's probably just personal preference but this way it makes more sense to me.

Reply:
Not possible due to evaluation order of decorators (bottom up)

if command_log is evaluated after the bot decorator everything burns down

Reply:
![fire_elmo.gif](https://media2.giphy.com/media/e5kbmb3wX3J1S/200.gif)